#! /bin/sh -e

# DP: Find header file for Boehm garbage collector.

if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch $pdir -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch $pdir -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

*** libobjc/Makefile.in~	Tue Nov 10 15:25:36 1998
--- libobjc/Makefile.in	Tue Nov 10 13:16:08 1998
***************
*** 65,71 ****
    -I$(srcdir)/$(MULTISRCTOP)../gcc/config -I$(MULTIBUILDTOP)../../gcc \
    -I$(srcdir)/$(MULTISRCTOP)../include
  
! OBJC_GCFLAGS=-DOBJC_WITH_GC=1
  OBJC_THREAD_FILE=thr-@OBJC_THREAD_FILE@
  
  .SUFFIXES:
--- 65,71 ----
    -I$(srcdir)/$(MULTISRCTOP)../gcc/config -I$(MULTIBUILDTOP)../../gcc \
    -I$(srcdir)/$(MULTISRCTOP)../include
  
! OBJC_GCFLAGS=-I/usr/include/gc -DOBJC_WITH_GC=1
  OBJC_THREAD_FILE=thr-@OBJC_THREAD_FILE@
  
  .SUFFIXES:
